compiler/rustc_typeck/src/collect.rs

```diff
@@ -3129,6 +3129,7 @@ fn codegen_fn_attrs(tcx: TyCtxt<'_>, did: DefId) -> CodegenFnAttrs {
                         E0535,
                         "invalid argument"
                     )
+                    .help("valid inline arguments are `always` and `never`")
```

Comment:
This could also emit two suggestions (not machine applicable), one for each variant. Not sure how the diagnostic would look, but if you want to try it out... If not, the PR lgtm, we can merge it as is.

Reply:
Maybe keep it as is for now as I want the style of this diagnostics to be the same as the invalid `repr` attribute argument one, shown below:

```
error[[E0552]](https://doc.rust-lang.org/nightly/error-index.html#E0552): unrecognized representation hint
 --> src/main.rs:1:8
  |
1 | #[repr(unknown)]
  |        ^^^^^^^
  |
  = help: valid reprs are `C`, `align`, `packed`, `transparent`, `simd`, `i8`, `u8`, `i16`, `u16`, `i32`, `u32`, `i64`, `u64`, `i128`, `u128`, `isize`, `usize`
```